mocks/mocks.py

```diff
@@ -15,29 +15,37 @@
 
 import ccdproc
 
-import pygadgetreader
+import pynbody
 
 import gunagala
 from gunagala.utils import ensure_unit
 
 from mocks.utils import load_yaml_config
 
 
-def read_gadget(data_path,
-                particle_type):
-    particle_pos = pygadgetreader.readsnap(data_path,
-                                           'pos',
-                                           particle_type)
-    particle_value = pygadgetreader.readsnap(data_path,
-                                             'mass',
-                                             particle_type)
-    sim_info = dict()
-    sim_info['hubble_constant'] = pygadgetreader.readheader(data_path, 'h')
-    sim_info['Omega_Lambda'] = pygadgetreader.readheader(data_path, 'Ol')
-    sim_info['Omega_0'] = pygadgetreader.readheader(data_path, 'O0')
-    sim_info['redshift'] = pygadgetreader.readheader(data_path, 'redshift')
-    sim_info['boxsize'] = pygadgetreader.readheader(data_path, 'boxsize')
-    return particle_pos, particle_value, sim_info
+def read_gadget(data_path):
```

Comment:
Since the simulations files are too big, it is not a good idea to add them to the repository. I will appreciate any suggestion about writing tests without uploading data.

Reply:
Does `pynbody` allow you to open a gadget file, edit it and then write back out a gadget file?

Reply:
I have not seen that feature in `pynbody` yet. But keep looking.

Reply:
Can do this via [`git-lfs`](https://git-lfs.github.com/)

Reply:
If you need a test dataset, let's ask the simulation folks for a tiny simulation run from e.g. https://github.com/sbird/MP-Gadget3/tree/master/examples/small . The gadget files we've been given are not public yet, so let's not host them here. Make a new Issue to add a test of `read_gadget` later on. Can close this PR before then.